dora/core/server/worker/src/main/java/alluxio/worker/modules/DoraWorkerModule.java

```diff
@@ -75,6 +75,10 @@ protected void configure() {
     bind(UfsManager.class).to(DoraUfsManager.class).in(Scopes.SINGLETON);
     bind(AlluxioConfiguration.class).toProvider(() -> Configuration.global());
 
+    // Create FileSystemContext shared across all worker components
+    FileSystemContext fileSystemContext = FileSystemContext.create();
+    bind(FileSystemContext.class).toInstance(fileSystemContext);
```

Comment:
Similar to the change in PagedDoraWorker constructor, I'm just moving some resource creation out of `PagedDoraWorker()` into this module definition, and injecting into the constructor. The code logic effectively doesn't change.